modules/local/payload/vcf/resources/usr/bin/main.py

```diff
@@ -165,14 +165,16 @@ def main(args):
             # Join the first two parts if there are multiple parts
             new_tool = "_".join(new_tool_parts[:2]) if len(new_tool_parts) > 1 else new_tool_parts[0]
             updated_pipeline_info[new_tool]=version
+    if seq_experiment_analysis_dict.get('analysis_tools (tools and versions)'):
+        tools_dict = dict(tool.split(' ', 1) for tool in seq_experiment_analysis_dict.get('analysis_tools (tools and versions)').split(', '))
+        updated_pipeline_info['FoundationOneCDx'] = tools_dict
```

Comment:
We need to use the `workflow_name` instead of hardcoded it to be `FoundationOneCDx`.